.github/CODEOWNERS

```diff
@@ -114,6 +114,7 @@ receiver/kafkametricsreceiver/                       @open-telemetry/collector-c
 receiver/kafkareceiver/                              @open-telemetry/collector-contrib-approvers @pavolloffay @MovieStoreGuy
 receiver/kubeletstatsreceiver/                       @open-telemetry/collector-contrib-approvers @pmcollins @dmitryax
 receiver/memcachedreceiver/                          @open-telemetry/collector-contrib-approvers @djaglowski
+receiver/mongodbreceiver/                            @open-telemetry/collector-contrib-approvers @djaglowski @schmikei
```

Comment:
This is something we should definitely discuss during the SIG meeting. I would expect the original contributors to be code owners, as they should be responsible for handling bugs.

Reply:
The name can be added, it will not have any effect, but at least we "manually" track the approval, which we do anyway for the moment.

Adding them to the org is a natural next step after the component is merged.

Reply:
@schmikei, based on this conversation, you should re-add your handle here.